source/containers.tex

```diff
@@ -462,7 +460,7 @@
 
 \indexcont{operator=}%
 \begin{itemdecl}
-r = a
+b = a
```

Comment:
This seems to be a pre-existing wording bug, as mentioned by @zygoloid : We don't require that "b" is an lvalue, which means this must work for xvalues, too. And that, effectively, means that operator= cannot be ref-qualified. Is that the intent?

Reply:
I think the current proposed resolution for https://wg21.link/lwg3028 fixes that. The commit msg for c69af479dacad40d86a8d2429f69cd302fc77047 refers to that issue.

Reply:
So, we're now intentionally creating a merge conflict between LWG3028 and this editorial issue? Could we at least advance LWG3028 expeditiously?

Reply:
Any news on this?

Reply:
@jwakely Ping?

Reply:
I'm resolving this thread, because I've force-pushed a new revision without the conflicts.